### PR TITLE
Add `asking orders` step

### PR DESCRIPTION
**Pull request by zheileman**
This step let the applicant choose what the court needs to do about
the children. When `other` is selected, it goes to another step to
enter more details.

### CODE REVIEW

File: lib/extensions/string_extension.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+module StringExtension
+  def true?
+    /(true|t|yes|y|1)$/i.match(to_s).present?
+  end
+
+  def false?
+    to_s.strip.empty? || /(false|f|no|n|0)$/i.match(to_s).present?
+  end
+
+  def to_bool
```
Comment (tatyree) [inline lib/extensions/string_extension.rb]: This isn't being used anywhere.  Do you need it at this point?

Comment (zheileman) [reply lib/extensions/string_extension.rb]: Hmm, not really. I guess I can just leave the true? / false? methods. I will remove it and update the tests as at the moment to_bool is the method being tested than implicitly uses the others.

File: lib/extensions/string_extension.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+module StringExtension
+  def true?
+    /(true|t|yes|y|1)$/i.match(to_s).present?
+  end
+
+  def false?
```
Comment (tatyree) [inline lib/extensions/string_extension.rb]: While it makes sense for logical completeness, I've just realised `.false?` isn't being used anywhere except the tests, either...

I'd suggest it should come out as well–makes the tests simpler and keeps potential mutants at bay 😄

Comment (zheileman) [reply lib/extensions/string_extension.rb]: Ok. I would normally leave it there for the logical sense as you say, but probably in this service will never be used, as looking at the decision trees we do things when something is checked, not when something is *not* checked. Or we cover these under the If (checked) else (not checked) statement. So will remove it 👍

**Pull request merged**